cli/cmd/quickStart.go

```
@@ -0,0 +1,110 @@
+		if err != nil {
+			fmt.Println("Failed to fetch images:", err)
+			return
+		}
+
+		var selectedStacks []string
+		promptStacks := &survey.MultiSelect{
+			Message:  "Select stacks to build:",
+			Options:  imageNames,
+			PageSize: 8,
+		}
+		survey.AskOne(promptStacks, &selectedStacks)
```

golangci-lint (1.64.8)

Error return value of `survey.AskOne` is not checked

(errcheck)